examples/inline_bot.rs

```diff
@@ -22,6 +22,7 @@ fn main() {
     let bot = RcBot::new(lp.handle(), &env::var("TELEGRAM_BOT_KEY").unwrap()).update_interval(200);
 
     let stream = bot.get_stream()
+        .unwrap()
```

Comment:
Or the stream could result in an error if the creation of the timer failes

Reply:
done